.github/workflows/build.yml

```diff
@@ -401,9 +401,13 @@ jobs:
       - name: Fetch test environment
         id: fetch
         if: needs.file-check.outputs.run == 'true'
-        uses: actions/download-artifact@v4
+        uses: Wandalen/wretry.action@v1
         with:
-          name: ${{ matrix.artifact_key }}-test-env
+          action: actions/download-artifact@v4
+          with: |
+            name: ${{ matrix.artifact_key }}-test-env
```

Comment:
```suggestion
          with: |
            name: ${{ matrix.artifact_key }}-test-env
            path: .
```

`path` here too?

Reply:
This one seems to work without it, but we probably should put it here just for consistency.